### PR TITLE
Sample test

### PR DESCRIPTION
**Pull request by kingwangzzang1234**

### CODE REVIEW

File: toreview.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# sample code
+
+```python
+from random import randint
+
```
Comment (ulgoon) [inline toreview.md:5]: PEP8 Well done!

File: toreview.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+def num_picker(num):
+    result = randint(1,num+1)
```
Comment (ulgoon) [inline toreview.md:8]: It's too simple. make this more complicated but awesome!!

**Review by ulgoon — CHANGES_REQUESTED** [review]

File: toreview.md (hunk 1 above)
Comment (kingwangzzang1234) [reply toreview.md:5]: Thanks

File: toreview.md (hunk 2 above)
Comment (kingwangzzang1234) [reply toreview.md:8]: Urgh... Okay..

File: toreview.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ from random import randint
 
 
 def num_picker(num):
-    result = randint(1,num+1)
-    return result
+    start_num = 1
+    if num >= 1 and num < 46:
+        end_num = num + 1
```
Comment (ulgoon) [inline toreview.md:10]: Why end_num is here???

**Review by ulgoon — APPROVED** [review]

**Pull request merged**